applications/osb-portal/src/pages/WorkspacePage.tsx

```diff
@@ -231,8 +245,13 @@ export const WorkspacePage = (props: any) => {
           </Box>
         </Box>
 
+          {
+            /*
+              Bottom: workspace resource panel and thumbnail
+            */
+          }
         <Box className={`verticalFit ${classes.workspaceResourcesInformation}`} display="flex" flexDirection="row">
-          <WorkspaceInteractions workspace={workspace} open={true} user={props.user} refreshWorkspace={() => { setRefresh(!refresh) }} openResource={handleResourceClick} />
+          <WorkspaceInteractions workspace={workspace} open={true} user={props.user} refreshWorkspace={() => { setRefresh(!refresh) }} openResource={handleResourceClick}  cloneWorkspace={props.cloneWorkspace}/>
```

Comment:
no need to pass it here actually. But you were right to put it here in the beginning as WorkspaceInteractions was not imported correctly from the component/index.ts file here. I fixed that today with the delete issue